rocks-bin/src/pin.rs

```diff
@@ -9,17 +9,17 @@ use rocks_lib::{
 };
 
 #[derive(Args)]
-pub struct Pin {
+pub struct ChangePin {
     package: RemotePackage,
 }
 
-pub fn pin(data: Pin, config: Config) -> Result<()> {
+pub fn pin(data: ChangePin, config: Config, pin: bool) -> Result<()> {
```

Comment:
issue: This feels confusing to read: a `pin` function that takes a `pin: bool` argument.

Can we split it into separate functions for pinning and unpinning, or pass in an enum to avoid boolean blindness?

Reply:
Funnily enough that was my first instinct, but it duplicated code unnecessarily and it felt worse than adding a boolean switch here. I was struggling with an alternative name: `repin`, `set_pin_state`? They're all pretty messy. I'll wrap the bool in an enum and also make other parts of the codebase use this enum for pinning.

Reply:
I think `set_pinned_state` is a good name (and I still prefer enums over booleans for readability).

Reply:
Let me know if there's anything you'd change with what I just pushed :)

Reply:
lgtm